src/parser/parsers.h

```diff
@@ -2130,10 +2130,32 @@ makeTableCopy(Ctx& ctx, Index pos, const std::vector<Annotation>& annotations) {
 template<typename Ctx>
 Result<>
 makeTableInit(Ctx& ctx, Index pos, const std::vector<Annotation>& annotations) {
+  // Note: binary and text formats for `table.init` are different. In the text
+  // format the table index is optional (with 0 as the default), and it comes
+  // after the elem index.
```

Comment:
This seems inconsistent with the code below that parses the table index first.

Reply:
The code is right. I updated the comment.